components/dashboard/src/components/Tooltip.tsx

```diff
@@ -9,6 +9,7 @@ import { useState } from 'react';
 export interface TooltipProps {
     children: React.ReactChild[] | React.ReactChild;
     content: string;
+    isProjectName?: boolean;
```

Comment:
Can we rename this to `allowWrap`  or similar?

Reply:
Good suggestion.